prism

```diff
@@ -0,0 +1 @@
+./packages/http/src/__tests__/fixtures/extensions/
```

Comment:
I've put a relative symlink in the root of the project folder to test this use case. https://github.com/stoplightio/prism/blob/529171ec862891f798c4a94781fb63ea099891ad/packages/http/src/mocker/__tests__/HttpMocker.spec.ts#L226

I'm not all that sure of this being a good solution for this problem.